python/ray/tune/tune.py

```diff
@@ -445,6 +447,32 @@ class and registered trainables.
             DeprecationWarning,
         )
 
+    error_message_map = (
+        {
+            "entrypoint": "Tuner(...)",
+            "search_space_arg": "param_space",
+            "restore_entrypoint": 'Tuner.restore(path="{path}", trainable=...)',
+        }
+        if _tuner_api
+        else {
+            "entrypoint": "tune.run(...)",
```

Comment:
When does an user normally call tune.run()?

Reply:
It's the old Tune API that we will deprecate at some point in the future.

Reply:
Got it.